nextcord/interactions.py

```diff
@@ -175,15 +219,23 @@ def _from_data(self, data: InteractionPayload):
                 pass
 
     @property
-    def client(self) -> Client:
-        """:class:`Client`: The client that handled the interaction."""
+    def client(self):
+        """:class:`Client`: Returns the client that handled the interaction. An alias exists under ``bot``."""
         return self._state._get_client()
+    
+    @property
+    def bot(self):
+        """:class:`Client`: Returns the client that handled the interaction. An alias exists under ``client``."""
+        return self.client
```

Comment:
This should just be `client` (see https://github.com/nextcord/nextcord/pull/358#discussion_r789219058 by Epic)

```suggestion
    def client(self) -> Client:
        """:class:`Client`: The client that handled the interaction."""
        return self._state._get_client()
```

Reply:
Ah, got it